packages/Manager/src/main/kotlin/nz/govt/eop/tasks/MinimumFlowLimitBoundariesGenerator.kt

```diff
@@ -24,12 +24,7 @@ class MinimumFlowLimitBoundariesGenerator(@Autowired val context: DSLContext) {
   @SchedulerLock(name = "checkMinimumFlowLimitBoundaries")
   @Transactional
   fun checkMinimumFlowLimitBoundaries() {
-    logger.debug { "Start task MinimumFlowLimitBoundariesGenerator" }
-    val needsRefresh = doesDataNeedRefresh()
-    if (needsRefresh) {
-      refresh()
-    }
-    logger.debug { "End task MinimumFlowLimitBoundariesGenerator" }
+    processDataRefresh(logger, "checkMinimumFlowLimitBoundaries", ::doesDataNeedRefresh, ::refresh)
```

Comment:
Its a kotlin thing that we can't share a constant for these names "checkMinimumFlowLimitBoundaries"